src/test/java/com/easypost/AddressTest.java

```diff
@@ -120,7 +126,13 @@ public void testCreateVerify() throws EasyPostException {
      */
     @Test
     public void testCreateAndVerify() throws EasyPostException {
-        Address address = Address.createAndVerify(Fixture.incorrectAddressToVerify());
+        List<String> verificationList = new ArrayList<>();
+        verificationList.add("delivery");
+
+        Map<String, Object> addressData = Fixture.incorrectAddressToVerify();
+        addressData.put("verify", verificationList);
```

Comment:
For this method, we don't need to pass the `verify` param because this endpoint will automatically verify the address passed to it.